src/sklearn_knn/_euclidean.py

```diff
@@ -1,14 +1,11 @@
-from sklearn.pipeline import Pipeline
+from ._base import MyStandardScaler, IDNeighborsClassifier
 
-from ._base import MyStandardScaler, IDNeighborsClassifier, KnnPipeline
 
-
-class Euclidean(KnnPipeline):
-    def _get_pipeline(self):
-        scaler = MyStandardScaler()
-        classifier = IDNeighborsClassifier(
-            n_neighbors=self.n_neighbors,
-            weights=self.weights,
-        )
-        steps = [("scaler", scaler), ("classifier", classifier)]
-        return Pipeline(steps)
+class Euclidean(IDNeighborsClassifier):
+    def fit(self, X, y):
+        X = MyStandardScaler().fit_transform(X)
+        return super().fit(X, y)
+    
+    def predict(self, X):
+        X = MyStandardScaler().fit_transform(X)
+        return super().predict(X)
```

Comment:
Yeah, I was hesitant to add that complexity just to cut a few lines of code, but I think it'll probably be worth it in the long run as we add the rest of the estimators.

I don't have a totally clear idea of how the mixin would work, but I imagine it would be more similar to your original implementation where each estimator would have a `_get_pipeline` method, which I think would allow us to continue passing args like `spp` to the transformer without having to deal with the crazy double-underscore pipeline argument syntax. I suppose if the pipeline is only ever just a transformer and an estimator, we could also consider avoiding pipelines altogether and just having the mixin run a transformer accessed through `_get_transformer`.